src/server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const pathname = normalizedURL.pathname;
+
+  if (!pathname.startsWith('/file/')) {
+    res.writeHead(400, 'Bad request');
+    res.end('Please enter valid file path, e.g. "/file/index.html"');
```

Comment:
When you close a request (send a response back to frontend) always add a return statement.